server/server.js

```diff
@@ -17,7 +16,7 @@ module.exports = function(port, googleAuthoriser) {
                 console.log("success");
                 adminToken = token;
                 res.cookie("sessionToken", token);
-                res.header("Location", "/dash.html");
+                res.header("Location", "/admin/dash.html");
```

Comment:
Cool, I can see that in Stephen's pull request. You could add it in this one, or just ignore the error, since moving the dash page is going to break functionality.